src/k8sconfiguration/azext_k8sconfiguration/custom.py

```diff
@@ -0,0 +1,176 @@
+    if cluster_scoped is None:
+        operator_scope = 'namespace'
+
+    # Determine operatorInstanceName
+    if operator_instance_name is None:
+        operator_instance_name = source_control_configuration_name
+
+    # Create helmOperatorProperties object
+    helm_operator_properties = HelmOperatorProperties()
+
+    if enable_helm_operator:
+        helm_operator_properties.chart_version = helm_operator_version.strip()
+        helm_operator_properties.chart_values = helm_operator_params.strip()
+    else:
+        helm_operator_properties = None
+
+    # Create sourceControlConfiguration object
+    source_control_configuration = SourceControlConfiguration(repository_url=repository_url,
+                                                              operator_namespace=operator_namespace,
+                                                              operator_instance_name=operator_instance_name,
+                                                              operator_type=operator_type,
+                                                              operator_params=operator_params,
+                                                              operator_scope=operator_scope,
+                                                              enable_helm_operator=enable_helm_operator,
+                                                              helm_operator_properties=helm_operator_properties)
+
+    # Try to create the resource
+    return client.create_or_update(resource_group_name, cluster_rp, cluster_type, cluster_name,
+                                   source_control_configuration_name, api_version, source_control_configuration)
+
+
+def update_k8sconfiguration(client, resource_group_name, cluster_name, name,
+                            configuration_type='sourceControlConfiguration', cluster_type='connectedClusters',
+                            repository_url=None, operator_params=None, enable_helm_operator=None,
+                            helm_operator_version=None, helm_operator_params=None, api_version='2019-11-01-preview'):
+    # Determine ClusterRP
+    if cluster_type.lower() == 'connectedclusters':
+        cluster_rp = 'Microsoft.Kubernetes'
+    elif cluster_type.lower() == 'managedclusters':
+        cluster_rp = 'Microsoft.ContainerService'
+    else:
+        raise CLIError("Invalid cluster-type.  Supported values are 'connectedClusters' and 'managedClusters'.")
+
+    # Validate configurationType
+    if configuration_type.lower() != 'sourcecontrolconfiguration':
+        raise CLIError('Invalid configuration_type.  Valid value is "sourceControlConfiguration"')
+
+    source_control_configuration_name = name.strip()
+
+    # Ensure apiVersion value
+    if api_version is None:
+        api_version = client.api_version
+
+    config = client.get(resource_group_name, cluster_rp, cluster_type, cluster_name,
+                        source_control_configuration_name, api_version).as_dict()
+
+    updateYes = False
```

Comment:
`updateYes` is not Pythonic, please use `update_yes`

Reply:
Fixed